src/components/cqlLibraryList/CqlLibraryList.tsx

```diff
@@ -28,8 +47,37 @@ const INITIAL_DELETE_DRAFT_STATE = {
   cqlLibrary: null,
 };
 
-export default function CqlLibraryList({ cqlLibraryList, onListUpdate }) {
+function IndeterminateCheckbox({
```

Comment:
can we have a unit test for this component. there is significant drop is coverage. rest looks good to me.